homeassistant/components/litejet/light.py

```diff
@@ -54,6 +61,11 @@ def name(self):
         """Return the light's name."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique identifier for this light."""
+        return str(self._index)
```

Comment:
If it makes sense to allow multiple entries, the unique id should include the config entry id or other unique identifier to ensure uniqueness between entries.